bundles/org.openhab.binding.daikin/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -5,16 +5,28 @@
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="thing-type:daikin:config">
-		<parameter name="host" type="text" required="true">
-			<label>Host</label>
-			<description>The Host address of the Daikin AC unit.</description>
-			<context>network_address</context>
-		</parameter>
-		<parameter name="refresh" type="integer" required="false">
-			<label>Refresh Interval</label>
-			<description>The number of seconds between fetches of the AC unit state.</description>
-			<default>60</default>
-		</parameter>
+			<parameter name="host" type="text" required="true">
+				<label>Host</label>
+				<description>The Host address of the Daikin AC unit.</description>
+				<context>network_address</context>
```

Comment:
```suggestion
				<context>network-address</context>
```

Reply:
this was inherited - will change it in the next commit